app/models/assessment.rb

```diff
@@ -168,7 +170,7 @@ def effective_version_threshold
   # If version_threshold == -1 (i.e. unlimited submissions without penalty)
   # or version_penalty == 0.0, no version penalty needs to be applied.
   def version_penalty?
-    effective_version_threshold > -1 && effective_version_penalty.value != 0.0
+    effective_version_threshold > -1 && effective_version_penalty.value.to_d != 0.0.to_d
```

Comment:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/FloatComparison
Chose to use bigdecimal